packages/manager/modules/support/src/tickets/tickets.module.js

```diff
@@ -0,0 +1,37 @@
+import angular from 'angular';
+
+import ovhUxNgOvhOtrs from '@ovh-ux/ng-ovh-otrs';
+import ovhUxNgTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouterAngularJs from '@uirouter/angularjs';
+import angularTranslate from 'angular-translate';
+import 'ovh-ui-angular';
+
+import 'ovh-ui-kit/dist/oui.css';
+
+import ticket from './ticket';
+
+import component from './tickets.component';
+import { registerState } from './tickets.routing';
+import ticketMessageService from './ticket/message/message.service';
+import ticketService from './ticket/ticket.service';
+import ticketsService from './tickets.service';
+
+const moduleName = 'ovhManagerSupportTickets';
+
+angular
+  .module(moduleName, [
+    angularTranslate,
+    'oui',
+    ovhUxNgOvhOtrs,
+    ovhUxNgTranslateAsyncLoader,
+    ticket,
+    uiRouterAngularJs,
+  ])
+  .component(component.name, component)
+  .config(registerState)
+  .run(/* @ngTranslationsInject:json ./translations */)
+  .service('ticketService', ticketService)
+  .service('ticketMessageService', ticketMessageService)
+  .service('ticketsService', ticketsService);
```

Comment:
Looks like quite a few services here. Would it make sense to merge `ticketsService` with `ticketService` ? They both handle `Ticket` elements right

Reply:
`tickets` works on the `tickets` view, `ticket` on the `Ticket` class